clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/view/GoogleLoadBalancerProvider.groovy

```diff
@@ -53,21 +53,32 @@ class GoogleLoadBalancerProvider implements LoadBalancerProvider<GoogleLoadBalan
 
   @Override
   Set<GoogleLoadBalancerView> getApplicationLoadBalancers(String application) {
-    def pattern = Keys.getLoadBalancerKey("*", "*", "${application}*")
-    def identifiers = cacheView.filterIdentifiers(LOAD_BALANCERS.ns, pattern)
-
-    def applicationServerGroups = cacheView.getAll(
+    String pattern = Keys.getLoadBalancerKey("*", "*", "${application}*")
```

Comment:
Is there a qualifier that could be put after the application name (like ':' or '-'). I don't know enough about the cache yet to know if that's feasible...

Reply:
I *think* the application LBs will always have the naming format `{application_name}-{your_stack}-{your-detail}` but if you don't add a stack or detail, then it will just have the same name as the application, so adding a qualifier will mean we don't find those.